atd-kits/signal_status_publisher.py

```diff
@@ -33,6 +33,11 @@
 SIGNAL_STATUS_RESOURCE_ID = "5zpr-dehc"
 SIGNALS_RESOURCE_ID = "p53x-x73x"
 
+STATUS_NAMES = {
+    1: "Scheduled flash",
+    2: "Unscheduled (Conflict) flash",
+    3: "Communication issue",
+}
```

Comment:
@ChristinaTremel here are the status code text I used, not a problem if you want something different